examples/complex/Main.lhs

```diff
@@ -31,9 +31,13 @@ import qualified Data.HashMap.Strict as HM
 import           Data.Maybe (isJust)
 import           Data.Text (Text, pack)
 #ifdef ENABLE_OBSERVABLES
+#ifdef RUN_ProcObseverSTM
 import           Control.Monad (forM)
 import           GHC.Conc.Sync (atomically, STM, TVar, newTVar, readTVar, writeTVar)
+#endif
+#endif
```

Comment:
I'm not familiar with the details, but wouldn't it be possible to drop the outer `ifdef`?